rust/src/api/types.rs

```diff
@@ -120,7 +116,8 @@ impl From<WalletTxOut> for TxOut {
 #[frb(dart_metadata=("freezed"))]
 pub struct Balance {
     pub asset_id: String,
-    pub value: i64,
```

Comment:
the reason balance value is i64 is because in the list of transactions a send tx shows a negative `Balance`